api/src/main/java/org/pmiops/workbench/api/RuntimeController.java

```diff
@@ -310,14 +310,15 @@ private Runtime getOverrideFromListRuntimes(String googleProject) {
   }
 
   @Override
-  public ResponseEntity<EmptyResponse> deleteRuntime(String workspaceNamespace) {
+  public ResponseEntity<EmptyResponse> deleteRuntime(
+      String workspaceNamespace, Boolean deleteDisk) {
     DbWorkspace dbWorkspace = lookupWorkspace(workspaceNamespace);
     String firecloudWorkspaceName = dbWorkspace.getFirecloudName();
     workspaceAuthService.enforceWorkspaceAccessLevel(
         workspaceNamespace, firecloudWorkspaceName, WorkspaceAccessLevel.WRITER);
 
     leonardoNotebooksClient.deleteRuntime(
-        dbWorkspace.getGoogleProject(), userProvider.get().getRuntimeName());
+        dbWorkspace.getGoogleProject(), userProvider.get().getRuntimeName(), deleteDisk);
```

Comment:
deleteDisk will be null if you merge this PR, is that okay if we just pass null to Leo? 
Suggest to check Null, and pass false if that is null

Reply:
I think you miss the 2nd part "and pass false if that is null", not fail the request